### PR TITLE
Add a logo

### PR DESCRIPTION
**Pull request by trexfeathers**
I used Microsoft Office to draw these - hence the originals are in a `.xlsx` file. This is not ideal, but it would take much longer to work out how to achieve the same with PyVista / manual drawing / whatever.

Also not ideal is PNG rather than SVG format. Office can produce SVGs, but when 3D perspective is applied (as in this case) the image is actually just a raster underneath.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -100,9 +100,17 @@
...
 html_theme = "pydata_sphinx_theme"
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
+
+html_theme_options = {
+    "logo": {
+        "text": "ncdata documentation",
```
Comment (trexfeathers) [inline docs/conf.py:114]: The title previously automatically included the version, too. I expect we could bring this back if desired.

**Comment by pp-mo** [comment]
This is great stuff !
And very timely, since I'm hoping to push along proper docs pages quite soon.

Thanks for taking the trouble -- I'm going to add you as a contributor / maintainer (hopefully will unblock future RTD runs ?)

**Review by pp-mo — CHANGES_REQUESTED** [review]
At the very worst, this is a great start !

Can you inject this into the main-page README also ?
I think that will improve the look of the entire project, 
though I suspect it will make it even more desirable to write a better oneline project summary

**Comment by trexfeathers** [comment]
> At the very worst, this is a great start !
> 
> Can you inject this into the main-page README also ? I think that will improve the look of the entire project, though I suspect it will make it even more desirable to write a better oneline project summary

I think I have done this (unless I am misunderstanding). [ncdata/README.md at `5395b61` · pp-mo/ncdata (github.com)](https://github.com/pp-mo/ncdata/blob/5395b6119b3259a982e21841eb9814771ebcf4cc/README.md)

**Comment by pp-mo** [comment]
> I think I have done this
OK I think I failed to understand this, because I stupidly followed the README link from the docs build instead of looking at the README as modified here !

Definitely good enough for now, thanks @trexfeathers

**Review by pp-mo — APPROVED** [review]

File: docs/conf.py (hunk 1 above)
Comment (pp-mo) [reply docs/conf.py:114]: I'm not worried about this.
The version isn't well handed in the current RTD builds anyway.

**Pull request merged**